idea_town/frontend/static-src/app/lib/router.js

```diff
@@ -32,7 +32,7 @@ export default Router.extend({
     }
   },
 
-  // 'experiment' is a URL slug: for example, 'universal-search'
+  // 'experiment' is a URL slug: for example, 'universal_search'
```

Comment:
This actually doesn't have to do with django as we pull the slug out and match it from the experiments collection. We could easily convert hyphens to underscores in this process. Seems unnecessary for this.